tests/release/service/happy_path_with_deployment.go

```diff
@@ -16,7 +14,7 @@ import (
 	"os"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release-pipelines", "withDeployment", "HACBS"), func() {
+var _ = framework.ReleaseSuiteDescribe("happy_path_with_deployment", Label("release-service", "withDeployment"), func() {
```

Comment:
This change depends on @cuipinghuo one. Using `release-service` now prevents the test to run, isn't it? You can use `release` here and let Cuiping to change it in her PR.

Reply:
Updated. Thanks

Reply:
the label used to be "release-pipeline", should this goes to pipeline directory？